gossip/client_test.go

```diff
@@ -27,15 +27,14 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
-func init() {
-	// With the default gossip interval, some tests
-	// may take longer than they need.
-	*GossipInterval = 20 * time.Millisecond
-}
-
 // startGossip creates local and remote gossip instances.
 // The remote gossip instance launches its gossip service.
 func startGossip(t *testing.T) (local, remote *Gossip, lserver, rserver *rpc.Server) {
+	// With the default gossip interval, some tests
+	// may take longer than they need.
+	gossipInterval := 20 * time.Millisecond
```

Comment:
Better to make these:

    const gossipInterval = 20 * time.Millisecond
    const gossipBootstrap = ""